tests/publishTests.js

```diff
@@ -508,7 +553,7 @@ publishFixture.forEach(function (fixture) {
     var toStub = ["checkChanges", "getDevVersion", "setVersion", "pubImpl", "clean"];
     var stub = createStubs(publish, toStub);
     var moduleVersion = modulePackage.version;
-    var devVersion = moduleVersion + "-testVersion";
+    var devVersion = moduleVersion + "-testVersion+devName";
```

Comment:
I'm having trouble following what the "+" character does here. It doesn't appear to feature in the template string and so as far as I can see this string suffix is just arbitrary.